integrations/standalone/tests/pageobjects/Detail.ts

```diff
@@ -29,45 +29,45 @@ export class Detail {
     await expect(this.title).toHaveText(title);
   }
 
-  async isDataClass() {
+  async expectToBeDataClass() {
     await this.classType.expectToExist();
     await this.description.expectToExist();
     await this.annotations.expectToExist();
 
-    await this.name.expectToNotExist();
-    await this.type.expectToNotExist();
-    await expect(this.persistent).toHaveCount(0);
-    await this.comment.expectToNotExist();
+    await this.name.expectToBeHidden();

```

Comment:
I'm not sure if we should wrap all this expect methods...
I mean for some page objects which are more complex this makes maybe sense, but for simple ones?
I think I prefer that the page object also exposes the main locator which then can be used for such assertions

`await expect(this.name.locator).toBeHidden();`